.changeset/three-peaches-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
```

Comment:
Patch because this feature should already be present as [we document that](https://docs.astro.build/en/guides/content-collections/#defining-a-collection-schema)